### PR TITLE
Committer vote and set up document.

### PR DESCRIPTION
**Pull request by wu-sheng**
Hi @apache/skywalking-committers 

I initialized this PR, because the [CN blog of new committer setup](http://skywalking.apache.org/zh/blog/2019-09-12-apache-skyWalking-committer-apply-process.html) reminds me, we miss this important guide.

The document describe the detail process, duty, right of committer and PMC member, and how could be one of them.

Please review, I will keep this PR open at least 72 hours.

### CODE REVIEW

**Comment by arugal** [comment]
I want to join, but I don't have time to contribute now. I hope there will be an opportunity later

**Comment by wu-sheng** [comment]
@arugal Take youe time. The open source community is open always.

File: docs/en/guides/README.md [hunk 1]
```diff
@@ -15,14 +15,18 @@ follow the next step to subscribe.
...
 All the following channels are open to the community, you could choose the way you like.
 * Submit an [issue](https://github.com/apache/skywalking/issues)
 * Mail list: **dev@skywalking.apache.org**. Mail to `dev-subscribe@skywalking.apache.org`, follow the reply to subscribe the mail list.
 * [Gitter](https://gitter.im/openskywalking/Lobby)
 * QQ Group: 392443393
 
+## Become official Apache SkyWalking Committer
+The PMC will assess the contributions of every contributor, including, but not limited to, 
+code contributions, and follow the Apache guides to promote, vote and invite new committer and PMC member.
+Read [Become official Apache SkyWalking Committer](asf/committer.md) to get details.
```
[the remark this answers is not in the document]
Comment (wu-sheng) [reply docs/en/guides/README.md]: We are only maintaining English version. Chinese document is only maintained by volunteer. See pinned issue, it is already out of date. We don't have enough translators to make it official.

**Pull request merged**